src/test/java/io/strimzi/kafka/bridge/http/AdminClientTest.java

```diff
@@ -0,0 +1,133 @@
+                .send(ar -> {
+                    context.verify(() -> {
+                        assertThat(ar.succeeded(), is(true));
+                        HttpResponse<JsonArray> response = ar.result();
+                        assertThat(response.statusCode(), is(HttpResponseStatus.OK.code()));
+                        List<String> bridgeResponse = response.body().getList();
+                        assertThat(bridgeResponse, hasItem(topic));
+                    });
+                    context.completeNow();
+                });
+    }
+
+    @Test
+    void getTopicTest(VertxTestContext context) throws Exception {
+        String topic = "testGetTopic";
+        kafkaCluster.createTopic(topic, 2, 1);
+
+        JsonArray records = new JsonArray();
+        JsonObject json = new JsonObject();
+        json.put("value", "hello");
+        records.add(json);
+
+        JsonObject root = new JsonObject();
+        root.put("records", records);
+
+        CompletableFuture<Boolean> producer = new CompletableFuture<>();
+        producerService()
+                .sendRecordsRequest(topic, root, BridgeContentType.KAFKA_JSON_JSON)
+                .sendJsonObject(root, ar1 -> {
+                    context.verify(() -> {
+                        assertThat(ar1.succeeded(), is(true));
+                        producer.complete(true);
+                    });
+                });
+        producer.get(TEST_TIMEOUT, TimeUnit.SECONDS);
+
+        baseService()
+                .getRequest("/topics/" + topic)
+                .as(BodyCodec.jsonObject())
+                .send(ar -> {
+                    context.verify(() -> {
+                        assertThat(ar.succeeded(), is(true));
+                        HttpResponse<JsonObject> response = ar.result();
+                        assertThat(response.statusCode(), is(HttpResponseStatus.OK.code()));
+                        JsonObject bridgeResponse = response.body();
+                        assertThat(bridgeResponse.getString("name"), is(topic));
+                        JsonArray partitions = bridgeResponse.getJsonArray("partitions");
+                        assertThat(partitions.size(), is(2));
+                        for (int i = 0; i < 2; i++) {
+                            JsonObject partition = partitions.getJsonObject(i);
+                            assertThat(partition.getInteger("partition"), is(i));
+                            assertThat(partition.getInteger("leader"), is(1));
+                            JsonArray replicas = partition.getJsonArray("replicas");
+                            assertThat(replicas.size(), is(1));
+                            JsonObject replica0 = replicas.getJsonObject(0);
+                            assertThat(replica0.getInteger("broker"), is(1));
+                            assertThat(replica0.getBoolean("leader"), is(true));
+                            assertThat(replica0.getBoolean("in_sync"), is(true));
```

Comment:
missing "configs" section check which seems to be missing in the code anyway.

Reply:
added the configs retrieval and check one of its values.